docs/platform-engineers/openapi/overview.md

```diff
@@ -0,0 +1,162 @@
+---
+title: VelaUX OpenAPI
+---
+
+This section demo by local address, so you first prepare the VelaUX.
+
+```bash
+vela port-forward addon-velaux -n vela-system 8000:80
+```
+
+## Authentication and API Example
+
+### Login and get the token
+
+Before you call the OpenAPI, you should complete authentication. VelaUX OpenAPI only supports local accounts. The following example is with the default admin account.
+
+```bash
+curl -H Content-Type:application/json -X POST -d '{"username": "admin", "password":"VelaUX12345"}' http://127.0.0.1:8000/api/v1/auth/login
+```
+
+> `http://127.0.0.1:8000` This is demo address, you should replace it with the real address. If you changed the password, replace it with the real password.
+
+expected output:
+
+```json
+{
+ "user": {
+  "createTime": "0001-01-01T00:00:00Z",
+  "lastLoginTime": "0001-01-01T00:00:00Z",
+  "name": "admin",
+  "email": "barnett.zqg@gmail.com",
+  "disabled": false
+ },
+ "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiZ3JhbnRUeXBlIjoiYWNjZXNzIiwiZXhwIjoxNjU2NTE0ODcyLCJpc3MiOiJ2ZWxhLWlzc3VlciIsIm5iZiI6MTY1NjUxMTI3Mn0.IqRWglW5cYMn8KJcbkoG55os9g-YTo_9UWFvbpZBKiY",
+ "refreshToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6ImFkbWluIiwiZ3JhbnRUeXBlIjoicmVmcmVzaCIsImV4cCI6MTY1NjU5NzY3MiwiaXNzIjoidmVsYS1pc3N1ZXIiLCJuYmYiOjE2NTY1MTEyNzJ9.mUjXj1BQ2c3MaVyYnMYefgH2g8Y-swgjyCAzH-GbZu8"
+}
+```
+
+* accessToken: This is the token to request other APIs.
+* refreshToken: This is the token to refresh the accessToken, which will expire in an hour.
```

Comment:
can we configure the expire time duration? and how?

Reply:
Currently, this is a fixed time.